frontend/src/utils/video.ts

```diff
@@ -70,7 +70,9 @@ export async function getVideoFromRateLaterListForComparison(
   otherVideo: string | null,
   currentVideo: string | null
 ): Promise<string | null> {
-  const rateLaterResult = await UsersService.usersMeVideoRateLaterList(99, 0);
+  const rateLaterResult = await UsersService.usersMeVideoRateLaterList({
+    limit: 99,
+  });
```

Comment:
We could maybe keep offset parameter for more clarity

Reply:
Done